sqlmesh/core/context.py

```diff
@@ -384,6 +384,7 @@ def __init__(
         self.auto_categorize_changes = self.config.plan.auto_categorize_changes
         self.selected_gateway = gateway or self.config.default_gateway_name
         self.gateway_managed_virtual_layer = self.config.gateway_managed_virtual_layer
+        self.infer_python_package_requirements = self.config.infer_python_package_requirements
```

Comment:
Is there a point in having this top-level value?

Reply:
Yeah. I missing the point of having this value

Reply:
I did it to be consistent with `gateway_managed_virtual_layer`, since it's the same pattern. Would you like me to change both of them?

Reply:
Done